app/services/evss_claim_service.rb

```diff
@@ -5,6 +5,7 @@
 require 'evss/auth_headers'
 
 # EVSS Claims Status Tool
+# [wipn8923] verify that this is replaced by LH object
 class EVSSClaimService
```

Comment:
The Lighthouse equivalent of this created by the mobile team is [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/lighthouse/benefits_documents/service.rb); however, it's tailored to their use case and I didn't use it in my `SubmitUploads` implementation. But I actually didn't need to - our use case was a little simpler and I didn't need a complex service because of other work they did I leveraged. (More deets to come in my draft PR)